csm/core/controllers/login.py

```diff
@@ -70,4 +70,5 @@ async def post(self):
         await self.request.app.login_service.logout(session_id)
         # TODO: Stop any websocket connection corresponding to this session
         Log.info('Session ended')
+        Log.info(f"user_id: {self.request.session.credentials.user_id} successfully logged out")
```

Comment:
Remove the above log at line #72. Not required.

Reply:
removed